crates/rpc/src/handlers/state.rs

```diff
@@ -0,0 +1,41 @@
+use std::{str::FromStr, sync::Arc};
+
+use alloy_primitives::B256;
+use ream_consensus::deneb::beacon_state::BeaconState;
+use ream_storage::{
+    db::ReamDB,
+    tables::{Table, beacon_state::BeaconStateTable, state_root_index::StateRootIndexTable},
+};
+
+use crate::types::errors::ApiError;
+
+pub async fn get_state_from_id(state_id: String, db: Arc<ReamDB>) -> Result<BeaconState, ApiError> {
```

Comment:
```suggestion
pub async fn get_state_from_id(state_id: String, db: Arc<ReamDB>) -> Result<BeaconState, ApiError> {
```

state_id should be an enum not a string